### ISSUE TITLE
Ebert to evercam-server

### ISSUE THREAD

**Issue by ijunaid8088** [evercam/evercam-server#568]
- [x] Attach Ebert to evercam-server https://ebertapp.io/github/evercam/evercam-server (show issue badge in readme file)

- [ ] Fix the issues (All possible ones)

**Comment by marcoherbst** [comment]
I have a hunch we will end up not using Ebert, but I would like you, @ijunaid8088 , to lead in terms of looking at the info it has provided and refreshing our approach to how we do automated code checking. There are many nice things in Ebert, before we drop it (if we do) I'd like to know that we've given it proper consideration.

**Comment by ijunaid8088** [comment]
Okay am going fix the issues stated by Ebert, maximum possible ones. then I will make a decision on it.

**Comment by ijunaid8088** [comment]
As I am seeing Ebert, it just a combination of some Codelinters such as Credo, CodeClimate and RuboCop etc, You can add config file to tell it to use ruboCop or not, Use Credo or not, and so on, from total 32 issues 25 are those which are from credo, and few from CodeClimate (7) about finding TODO is code. 

![image](https://cloud.githubusercontent.com/assets/13561999/21010664/1f8f6e5a-bd6f-11e6-9c7e-4ba9f4a43e1e.png)

We are already using Credo. I don't think we need something else for automated code checking as If we are using Credo. 

In short Ebert will gather all codeLinters in one file as now it's using Credo and CodeClimate to judge the coding. 

Code climate is showing these as Bug Risk issues
![image](https://cloud.githubusercontent.com/assets/13561999/21010683/489694b8-bd6f-11e6-9530-64a22f4faa0c.png)

Eh? I don't think it's a "Bug Risk issue." Excluding those 7 all others are reported by Credo. 

I see no point in using Ebert anymore.

**Comment by marcoherbst** [comment]
ok, done

**Comment by ijunaid8088** [comment]
Right!

Am removing Ebert SVG from readmeMD file.
